menu-api-config.yaml

```diff
@@ -1,12 +1,12 @@
-WPVERITAS_URL: "https://wp-veritas-test.epfl.ch/api/v1/sites?platform_target=openshift-3"
-POD_NAME: "XXXX"
+WP_SERVICE_NAME: "172.24.0.6"
```

Comment:
This needs at least a comment explaining how to retrieve the container's IP.
(Of course, it would be even better to script this change e.g. from the Makefile)